packages/editor/src/client/panel/publishArticlePanel.tsx

```diff
@@ -45,29 +45,56 @@ export function PublishArticlePanel({
         )}
         <Form fluid={true}>
           <FormGroup>
-            <ControlLabel>{t('articleEditor.panels.publishDate')}</ControlLabel>
+            <ControlLabel>{t('articleEditor.panels.publishTime')}</ControlLabel>
             <DatePicker
-              block
+              style={{width: 100, marginRight: 8}}
+              placement="auto"
               value={publishDate}
-              format="YYYY-MM-DD HH:mm"
+              format="HH:mm"
               ranges={[
                 {
-                  label: 'Now',
+                  label: t('articleEditor.panels.now'),
+                  value: new Date()
+                }
+              ]}
+              onChange={publishDate => setPublishDate(publishDate)}
+            />
+            <DatePicker
+              style={{width: 130}}
+              placement="auto"
+              value={publishDate}
+              format="DD MMM YY"
+              ranges={[
+                {
+                  label: t('articleEditor.panels.now'),
                   value: new Date()
                 }
               ]}
               onChange={publishDate => setPublishDate(publishDate)}
```

Comment:
If i change the date after i changed the time the time will be reset to the current time. So you have to improve the `setPublishDate(...)`  function to include the time of the already set date.

This link might help. https://www.digitalocean.com/community/tutorials/understanding-date-and-time-in-javascript

Reply:
@roosnic1 can you have a look again to see if the time and dates are behaving separately? PR still needs some small changes, but would appreciate if you can check my approach meanwhile.